experimental/generation/composer-form-dialog.md

```diff
@@ -0,0 +1,61 @@
+# Form dialogs in Bot Framework Composer
+
+The form dialogs feature in Composer uses [dialog generation][generation] to automatically create a dialog from a form. The form dialog collects information for fields of the form while handling ambiguity, corrections, help, navigation and multiple fields at once. With form dialogs, Composer uses your form and creates the adaptive dialog, the bot responses, and the language model for you.
+
+**Note:** Form dialogs is a preview feature of Composer and should *not* be used in a production environment. To try an early version of form dialogs, follow the steps below.
+
+## Enabling the form dialogs preview
+
+To enable the form dialogs preview in Composer, follow these steps:
+
+1. Install the [Bot Framework Composer][link to Composer downloads].
+2. In Composer, go to your **Settings** by clicking on the gear icon at the bottom of the menu.
```

Comment:
A few things:

1. In the Composer docs do not use 'gear icon', so removing to be consistent.
1. Adding a step to explicitly explain how to run composer.
1. Combined steps 2 & 3. This is consistent with many other steps in Composer docs.
1. Specifying `navigation pane`, which is used consistently in Composer docs.
1. Changing numbering to use the standard of each number in the list being `1.`

```suggestion
1. Once installed, run Composer in your browser using the address http://localhost:3000.
1. Select **Settings** from the Composer menu, then **Application settings** in the navigation pane.
```